src/pps.mll

```diff
@@ -82,8 +95,12 @@
     Fmt.str "%s%s" (print_gospel `TwoAt start_p end_p s) (print l)
   | Other o :: Spaces sp :: Spec (start_p, end_p, s) :: l ->
     Fmt.str "%s%s%s%s" o sp (print_gospel `TwoAt start_p end_p s) (print l)
+  | Other o :: Spaces sp :: Documentation (doc_start_p, doc_end_p, d) :: Spaces sp' :: Spec (start_p, end_p, s) :: l ->
```

Comment:
That would be admittedly rather ugly, but the `Spaces` mean that
```
val f: int -> int(** doc *)(*@ spec *)
```
isn’t covered as one would hope.